tests/integration/telemetry/stats/prometheus/nullvm/accesslogs_test.go

```diff
@@ -51,15 +52,55 @@ func TestAccessLogs(t *testing.T) {
 		Features("observability.telemetry.logging").
 		Run(func(t framework.TestContext) {
 			t.NewSubTest("enabled").Run(func(t framework.TestContext) {
+				applyTelemetryResource(t, true)
 				runAccessLogsTests(t, true)
+				deleteTelemetryResource(t, true)
 			})
 			t.NewSubTest("disabled").Run(func(t framework.TestContext) {
+				applyTelemetryResource(t, false)
 				runAccessLogsTests(t, false)
+				deleteTelemetryResource(t, false)
 			})
 		})
 }
 
-func runAccessLogsTests(t framework.TestContext, expectLogs bool) {
+func TestAccessLogsDefaultProvider(t *testing.T) {
+	framework.NewTest(t).
+		Features("observability.telemetry.logging.defaultprovider").
+		Run(func(t framework.TestContext) {
+			t.NewSubTest("disabled").Run(func(t framework.TestContext) {
+				cfg := `
+accessLogFile: "/dev/null"
```

Comment:
a bit confused about /dev/null.. should it be `""` if we are testing disabled?

Reply:
fixed